docs/notes/2.28.x.md

```diff
@@ -46,7 +44,9 @@ Added internal support for executing a NodeJS tool where the executable name is
 
 #### Docker
 
-Added an option to [the `[docker]` subsystem](https://www.pantsbuild.org/prerelease/reference/subsystems/docker) to allow bypassing the `suggest_renames` functionality. There are currently situations where this code path can [introduce performance degradation](https://github.com/pantsbuild/pants/issues/22246), and this flag allows the end user to work around that issue.
+Added an option to [the `[docker]` subsystem](https://www.pantsbuild.org/2.28/reference/subsystems/docker) to allow
```

Comment:
Style-nit: I thought we allowed long lines for Markdown instead of manually wrapping? (and let the Markdown renderer wrap as needed)

Reply:
Whoops, reverting.